plugins/src/messages.rs

```diff
@@ -150,11 +150,31 @@ pub(crate) struct RpcMethod {
     pub(crate) usage: String,
 }
 
+#[derive(Serialize, Default, Debug, Clone)]
+pub struct NotificationTopic {
```

Comment:
It would be really cool to provide a procedural macro such that topics could be derived from an enum (we could then also set the payload there) but lets save that for another day ;)